setup.cfg

```diff
@@ -1,9 +1,37 @@
 [aliases]
-test=nosetests
+test = nosetests
 
 [nosetests]
-verbosity=3
-detailed-errors=1
-processes=1
-process-restartworker=1
-process-timeout=400
+cover-inclusive = true
+detailed-errors = 1
+nocapture = true
+process-restartworker = 1
+process-timeout = 400
+processes = 1
+verbosity = 3
+
+[flake8]
+max-line-length = 90
+max-doc-length = 90
+docstring-convention = numpy
+statistics = True
+show-source = True
+; Print the total number of errors.
+count = True
+; max-complexity = 10
+; List of checks to ignore
+; D401 First line should be in imperative mood; try rephrasing
+; W503: line break before binary operator (This is incompartible with Black,
+;       See: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-breaks-binary-operators)
+ignore = D401,E211,W503
```

Comment:
Why are you including `E211` in your list of checks to ignore here.

Reply:
Eish, copy and paste...it seems that some gremlins got into the system and this is the result.

Will fix it, good catch.